index.html

```diff
@@ -0,0 +1,237 @@
+              fill="white"
+            />
+          </svg>
+        </li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+    </section>
+    <nav class="navbar">
+      <img class="navbar-logo" src="./img/logo.png" alt="logo" />
+      <ul class="navbar__ul">
+        <li><a href="about.html">About</a></li>
+        <li><a href="#">Program</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#">Sponsor</a></li>
+        <li><a href="#">News</a></li>
+        <li class="li-special"><a href="#">CC Campaign</a></li>
+      </ul>
+    </nav>
+    <!-- Desktop view end -->
+
+    <section class="header">
+      <svg
+        class="header__svg"
+        xmlns="http://www.w3.org/2000/svg"
+        xmlns:xlink="http://www.w3.org/1999/xlink"
+        width="31"
+        height="28"
+        viewBox="0 0 31 28"
+        fill="none"
+      >
+        <rect width="31" height="27.28" fill="url(#pattern0)" />
+        <defs>
+          <pattern
+            id="pattern0"
+            patternContentUnits="objectBoundingBox"
+            width="1"
+            height="1"
+          >
+            <use xlink:href="#image0_68_6" transform="scale(0.02 0.0227273)" />
+          </pattern>
+
+          <image
+            id="image0_68_6"
+            width="50"
+            height="44"
+            xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADIAAAAsCAYAAAAn4+taAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAyRpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMy1jMDExIDY2LjE0NTY2MSwgMjAxMi8wMi8wNi0xNDo1NjoyNyAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvIiB4bWxuczp4bXBNTT0iaHR0cDovL25zLmFkb2JlLmNvbS94YXAvMS4wL21tLyIgeG1sbnM6c3RSZWY9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9zVHlwZS9SZXNvdXJjZVJlZiMiIHhtcDpDcmVhdG9yVG9vbD0iQWRvYmUgUGhvdG9zaG9wIENTNiAoTWFjaW50b3NoKSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDpBMEM2QUUyRjMzMDQxMUU1OUQyQ0NDMDJGMDc1OEMxOSIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDpBMEM2QUUzMDMzMDQxMUU1OUQyQ0NDMDJGMDc1OEMxOSI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkEwQzZBRTJEMzMwNDExRTU5RDJDQ0MwMkYwNzU4QzE5IiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOkEwQzZBRTJFMzMwNDExRTU5RDJDQ0MwMkYwNzU4QzE5Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+2Wl8LQAAAIJJREFUeNrs2EEKgzAUBNC0CK56uqB3qAeqd2iP58qV/uQKXQTj+zD7PBgYyCPnfKQO7pk6ORAQEBCQJjdE5sgY2S9qqG8vkJ9qgYCAgLTYkU/kFdkuaqhvL5C3aoGAgICA/DGIay+DuKgWCAgISIsdmVInH3Rf1QIBAQEBuTvkFGAAL5wOYApfVmAAAAAASUVORK5CYII="
+          />
+        </defs>
+      </svg>
+      <a href="index.html">
+        <img
+          class="header__img"
+          src="./img/logo.png"
+          alt="CC Global Summit 2015"
+        />
```

Comment:
- [ ] Great job adding the hamburger on the homepage. :+1: Kindly remove the logo image on the mobile version as suggested by the previous reviewers. ✍️


| Your design                                                             | Behance design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/226120734-0e07a2c4-d10a-40a0-82bf-8808fc6edbc7.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/226120740-3eba0c03-aaad-4450-a0a1-7009f8e564ce.png" width="300">|

Reply:
Hello @amejid,

Could you please check the figma's design? That feature is not present there, this is really frustrating to request to change something that is not in the design. **The logo must be present in both Home and About sections for mobile**

![image](https://user-images.githubusercontent.com/109441277/226121251-d6028dd4-9d5a-4d20-bc1b-5475851f8c2f.png)